test/test_utils.py

```diff
@@ -9,6 +9,12 @@
     get_valid_service_account_id_for_user,
 )
 
+# for testing aws
+import boto3
```

Comment:
try to merge these with the previous imports and keep the 3 standard sections, alphabetize within the sections